sympy/sets/sets.py

```diff
@@ -1808,6 +1836,43 @@ def __lt__(self, other):
         return self.is_proper_subset(other)
 
 
+class SymmetricDifference(Set):
+    """Representes the Symmetric Difference of a set with another set.
```

Comment:
Yes, but you could add citation of this page right here (with `[1]_` in an appropriate place).